gensim/test/test_sklearn_integration.py

```diff
@@ -105,7 +105,7 @@ def testPipeline(self):
         text_lda = Pipeline((('features', model,), ('classifier', clf)))
         text_lda.fit(corpus, data.target)
         score = text_lda.score(corpus, data.target)
-        self.assertGreater(score, 0.50)
+        self.assertGreater(score, 0.40)
```

Comment:
Split into a separate PR! Thank you so much. Taught me a lot about git and got me my first commit :)